src/lib/seam/components/AccessCodeTable/AccessCodeTable.tsx

```diff
@@ -140,9 +142,13 @@ export function AccessCodeTable({
       <ContentHeader onBack={onBack} />
       <TableHeader>
         <div className='seam-left'>
-          <TableTitle>
-            {t.accessCodes} <Caption>({filteredAccessCodes.length})</Caption>{' '}
-          </TableTitle>
+          {title != null ? (
+            <TableTitle>
+              {t.accessCodes} <Caption>({filteredAccessCodes.length})</Caption>{' '}
```

Comment:
Do we want to add another prop (in a separate PR) for hiding the device count as well?

Reply:
Interesting so if they override the title then there is no way to see device count. I think for this first version it's ok.

We could either allow the title prop to be a template literal that allows them to specify a placeholder for the count, or we could add a new prop onDevices

Reply:
> Interesting so if they override the title then there is no way to see device count. I think for this first version it's ok.

The opposite, actually. They can always see the device count unless `title === null`. I was just thinking ahead that some people _might_ wanna remove the count. We can wait until a customer asks for this though

Reply:
Exposing data related hooks as props probably needs more careful thought. I don't think we should add it adhoc here.

Reply:
Sounds good to me! We can push this to an issue 👍